tests/onnx/test_onnx_v2.py

```diff
@@ -230,6 +230,7 @@ def test_values_override(self):
     ("blenderbot", "facebook/blenderbot-400M-distill"),
     ("bigbird-pegasus", "google/bigbird-pegasus-large-arxiv"),
     ("longt5", "google/long-t5-local-base"),
+    ("pegasus", "google/pegasus-large"),
```

Comment:
Do you happen to know if there exists a smaller checkpoint on the Hub that can serve for testing?